### ISSUE TITLE
mouse/keyboard click does not work after some tab switch / window minimize

### ISSUE THREAD

**Issue by hamidb80** [zen-browser/desktop#4008]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

since recent update, after some time the mouse/keyboard does not work in "some" tabs, I can close tabs or switch between tabs using shortcuts and open Inspect-Elements, but clicking in tabs does work ( like it's freezing ) and the only way is to close the browser completely and open it again, very frustrating.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.5 (Firefox 133.0.3) (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by Stannieman** [comment]
Not sure if it's the exact same issue, but I experience this too on macOS.
It doesn't happen "after some time", but often after switching tabs, then the tab I go to suddenly cannot be interacted with.
The only thing that works is to close the tab and open a new one (or unloaded in case of pinned) to restore interaction. Just a refresh does not work.
I also feel that in 1.7b it has become a bit worse than in 1.6b, but that could just be my perception.

**Comment by hamidb80** [comment]
> The only thing that works is to close the tab and open a new one

In my case if switching back and forth via `ctrl+Tab` and `ctrl+shift+Tab` resolves the issue, please check if this works for you

**Comment by Stannieman** [comment]
No it doesn't. It does seem though that if it happens it can happen to a few tabs "at once" when switching between them.

**Comment by hamidb80** [comment]
related ? 
https://github.com/zen-browser/desktop/issues/4399

**Comment by Stannieman** [comment]
It sounds very similar indeed, and I think it's generic and not just related to essentials. Or maybe essentials and pinned tabs.
Anyway the symptoms are also slightly different than what I first thought. Actually when it happens, all tabs become unresponsive. And you don't need to close and reopen them, just wait and after a while everything works fine again. I need to try keep the task manager open to see if there is a CPU spike or something during the issue.

To get it "unlocked" it might be sufficient to open a new tab and wait until it's fully loaded before switching tabs again.
So TLDR: do not navigate away from a tab until it finished loading.

**Comment by hamidb80** [comment]
@Stannieman seems like it's been fixed in new version, please check

**Comment by Stannieman** [comment]
I have not encountered it anymore this weeken so I assume it is indeed fixed.
